src/olympia/reviewers/serializers.py

```diff
@@ -586,3 +580,16 @@ def validate(self, data):
                     'You can\'t submit a line number without associating '
                     'it to a filename.')})
         return data
+
+
+class DraftCommentSerializerReadOnly(DraftCommentSerializer):
+    version_id = serializers.SerializerMethodField()
```

Comment:
you can do `version_id = serializers.IntegerField(source='version.id')` directly if you don't need any additional processing in the `get....` method.

Reply:
Thanks!

Reply:
actually, doh, it's even easier - `version_id` is already a property of `DraftComment` (each foreign key is available as both `.object` and `.object_pk`)

Reply:
@eviljeff I made a change, but the tests are still failing. I imagine I need to make some sort of associated change to the view, but I can't seem to figure that out. Can you help?